python/py-tmuxp/Portfile

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=Portfile:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-tmuxp
+version             1.5.0a1
+categories-append   devel
+license             MIT
+platforms           darwin
+supported_archs     noarch
+
+maintainers         {@egorenar posteo.net:egorenar-dev} openmaintainer
+
+description         tmux session manager.
+long_description    ${description}
+
+homepage            https://github.com/tmux-python/tmuxp/
+master_sites        pypi:t/${python.rootname}/
+distname            ${python.rootname}-${version}
+python.versions     27 34 35 36 37
```

Comment:
Please remove the old py34 and py35 versions.

Reply:
fixed.